nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx

```diff
@@ -51,6 +52,21 @@ function CustomLeva() {
           value: true,
         });
       }, levaConf.settings);
+    } else if (levaConf.type === 'BUTTON_GROUP') {
+      // Add a button group.
+      leafFolder[key] = buttonGroup({
+        label: `Train Speed`,
+        hint: 'Select the training speed, affects viewer render quality, not final render quality',
```

Comment:
@tancik does this `hint` field do anything? It's not in the TypeScript definition, which only has `label` and `opts`: https://github.com/pmndrs/leva/blob/d658bf0b3cb129e11465e7079dcaed82882eb1bc/packages/leva/src/types/public.ts#L59-L64

Reply:
also, any reason to put `Train Speed` in backticks but the hint in single quotes?

Reply:
Here? https://github.com/pmndrs/leva/blob/d658bf0b3cb129e11465e7079dcaed82882eb1bc/packages/leva/src/types/public.ts#L147
No reason for different ticks

Reply:
Yeah, but the input of `buttonGroup` is just `ButtonGroupOpts`: https://github.com/pmndrs/leva/blob/d658bf0b3cb129e11465e7079dcaed82882eb1bc/packages/leva/src/helpers/buttonGroup.ts#L9-L11

And `ButtonGroupOpts` only has `label` and `opts` (as linked above)

=> I get an error when I try to assign `hint` to something:
<img width="460" alt="Screenshot 2023-05-15 at 11 19 14 AM" src="https://github.com/nerfstudio-project/nerfstudio/assets/6992947/475274a7-d65a-4799-872d-90f3da8e5dde">

Reply:
aghh, that logic doesn't belong in the js. Fixed now.